src/main/java/sirius/db/mixing/DateRange.java

```diff
@@ -276,17 +294,19 @@ public LocalDateTime getUntil() {
      */
     public <C extends Constraint> void applyTo(String field, boolean useLocalDate, Query<?, ?, C> query) {
         List<C> constraints = new ArrayList<>(2);
-        if (from != null) {
+        if (fromSupplier != null) {
```

Comment:
hier sollten auch die getFrom und getUntil aufgerufen werden und im if geprüft werden ob die resultierenden localdatetimes null sind. sonst ändert sich das verhalten durch den PR

Reply:
What could make such a supplier return null in first place?

Reply:
Dont know, but better safe than sorry. The null checks are already in place